### ISSUE TITLE
Warning: Unexpected input(s) 'target_commitish'

### ISSUE THREAD

**Issue by gandarez** [softprops/action-gh-release#105]
When passing a target commit it's showing a warning even if it's described in `action.yml`.

```
Warning: Unexpected input(s) 'target_commitish', valid inputs are ['body', 'body_path', 'name', 'tag_name', 'draft', 'prerelease', 'files']
```

**Comment by fsz-codeshop** [comment]
@gandarez Check which tag/version you are using.

"softprops/action-gh-release@v1" does not have the target_commitish, I got that same warning.

It looks like the field is only available @master

**Comment by jaylenw** [comment]
Using `@master` seemed to work for me.

**Comment by rohit-gohri** [comment]
@softprops Can we get a new release to use this feature without using `@master`?

**Comment by softprops** [comment]
released in [0.1.6](https://github.com/softprops/action-gh-release/blob/master/CHANGELOG.md#016) (also in the current v1 tag
